### ISSUE TITLE
Hotkeys activate if they overlap with one of the F3 button combos

### ISSUE THREAD

**Issue by itsTyrion** [Wurst-Imperium/Wurst7#837]
Before posting, please make sure that this hasn't already been reported by someone else.

## Description
As per title, hotkeys trigger if they overlap with an F3 action. 
For example, I have `say /fly` bound to G as hotkey for server commands (I have quite a few of those). 
When pressing F3+G to show chunk borders, it also triggers that hotkey

## Steps To Reproduce
Please make a list of steps to reproduce the problem (ideally from a fresh Wurst installation):
1. Make a hotkey that overlaps with any of the F3 button combos - F3 + (A/B/C/D/G/H/I/L/N/P/Q/S/T)
2. use the F3 action, it also triggers the hotkey

## Screenshots / Videos (Optional)
If applicable, add screenshots, videos or other files to help explain the problem.

## Crash Report
```
---- Minecraft Crash Report ----
// Sorry :(
Time: 2023-06-13 16:26:15
Description: Manually triggered debug crash
java.lang.Throwable: Manually triggered debug crash
	at net.minecraft.class_309.method_1474(class_309.java:506)
	at net.minecraft.class_310.method_1574(class_310.java:1955)
	at net.minecraft.class_310.method_1523(class_310.java:1180)
	at net.minecraft.class_310.method_1514(class_310.java:801)
	at net.minecraft.client.main.Main.main(Main.java:237)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:468)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:88)
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:126)
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:71)
A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------
-- Head --
Thread: Render thread
Stacktrace:
	at net.minecraft.class_309.method_1474(class_309.java:506)
-- Manual crash details --
Details:
	Modules: 
		OpenAL.dll:Main implementation library:1.21.1:
		RTSSHooks64.dll
		UMPDC.dll
		awt.dll:Zulu Platform x64 Architecture:19.0.2:Azul Systems Inc.
		extnet.dll:Zulu Platform x64 Architecture:19.0.2:Azul Systems Inc.
		glfw.dll:GLFW 3.4.0 DLL:3.4.0:GLFW
		java.dll:Zulu Platform x64 Architecture:19.0.2:Azul Systems Inc.
		javaw.exe:Zulu Platform x64 Architecture:19.0.2:Azul Systems Inc.
		jemalloc.dll
		jimage.dll:Zulu Platform x6
4 Architecture:19.0.2:Azul Systems Inc.
		jli.dll:Zulu Platform x64 Architecture:19.0.2:Azul Systems Inc.
		jna14561258249326312587.dll
		jna6747284960950064228.dll:JNA native library:6.1.4:Java(TM) Native Access (JNA)
		jsvml.dll:Zulu Platform x64 Architecture:19.0.2:Azul Systems Inc.
		jvm.dll:Zulu 64-Bit Server VM:19.0.2:Azul Systems Inc.
		lwjgl.dll
		lwjgl_opengl.dll
		lwjgl_stb.dll
		management.dll:Zulu Platform x64 Architecture:19.0.2:Azul Systems Inc.
		management_ext.dll:Zulu Platform x64 Architecture:19.0.2:Azul Systems Inc.
		mdnsNSP.dll:Bonjour Namespace Provider:3,0,0,10:Apple Inc.
		net.dll:Zulu Platform x64 Architecture:19.0.2:Azul Systems Inc.
		nio.dll:Zulu Platform x64 Architecture:19.0.2:Azul Systems Inc.
		nvoglv64.dll:NVIDIA Compatible OpenGL ICD:31.0.15.3141:NVIDIA Corporation
		sunmscapi.dll:Zulu Platform x64 Architecture:19.0.2:Azul Systems Inc.
		verify.dll:Zulu Platform x64 Architecture:19.0.2:Azul Systems Inc.
		zip.dll:Zulu Platform x64 Architecture:19.0.2:Azul Systems Inc.
Stacktrace:
	at net.minecraft.class_309.method_1474(class_309.java:506)
	at net.minecraft.class_310.method_1574(class_310.java:1955)
	at net.minecraft.class_310.method_1523(class_310.java:1180)
	at net.minecraft.class_310.method_1514(class_310.java:801)
	at net.minecraft.client.main.Main.main(Main.java:237)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:468)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:88)
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:126)
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:71)
-- Affected level --
Details:
	All players: 1 total; [class_746['IchKnuddelDich'/54470, l='ClientLevel', x=-3662.13, y=117.00, z=5135.80]]
	Chunk stats: 1024, 461
	Level dimension: minecraft:overworld
	Level spawn location: World: (-3,63,-8), Section: (at 13,15,8 in -1,3,-1; chunk contains blocks -16,-64,-16 to -1,319,-1), Region: (-1,-1; contains chunks -32,-32 to -1,-1, blocks -512,-64,-512 to -1,319,-1)
	Level time: 362740667 game time, 26539963 day time
	Server brand: Waterfall <- §5Purpur §0| §6Viel Spaß §4? §0|§f Never gonna give you up, never gonna let you down, never gonna run around and desert you.
	Server type: Non-integrated multiplayer server
Stacktrace:
	at net.minecraft.class_638.method_8538(class_638.java:455)
	at net.minecraft.class_310.method_1587(class_310.java:2406)
	at net.minecraft.class_310.method_1514(class_310.java:820)
	at net.minecraft.client.main.Main.main(Main.java:237)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:468)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:88)
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:126)
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:71)
-- Last reload --
Details:
	Reload number: 1
	Reload reason: initial
	Finished: Yes
	Packs: vanilla, fabric
-- System Details --
Details:
	Minecraft Version: 1.19.4
	Minecraft Version ID: 1.19.4
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 19.0.2, Azul Systems, Inc.
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode, sharing), Azul Systems, Inc.
	Memory: 393857656 bytes (375 MiB) / 1333788672 bytes (1272 MiB) up to 1468006400 bytes (1400 MiB)
	CPUs: 12
	Processor Vendor: AuthenticAMD
	Processor Name: AMD Ryzen 5 5600 6-Core Processor              
	Identifier: AuthenticAMD Family 25 Model 33 Stepping 2
	Microarchitecture: Zen 3
	Frequency (GHz): 3.50
	Number of physical packages: 1
	Number of physical CPUs: 6
	Number of logical CPUs: 12
	Graphics card #0 name: NVIDIA GeForce GTX 1070
	Graphics card #0 vendor: NVIDIA (0x10de)
	Graphics card #0 VRAM (MB): 4095.00
	Graphics card #0 deviceId: 0x1b81
	Graphics card #0 versionInfo: DriverVersion=31.0.15.3141
	Memory slot #0 capacity (MB): 8192.00
	Memory slot #0 clockSpeed (GHz): 3.20
	Memory slot #0 type: DDR4
	Memory slot #1 capacity (MB): 8192.00
	Memory slot #1 clockSpeed (GHz): 3.20
	Memory slot #1 type: DDR4
	Virtual memory max (MB): 31948.14
	Virtual memory used (MB): 25458.04
	Swap memory total (MB): 15638.54
	Swap memory used (MB): 2819.24
	JVM Flags: 20 total; -XX:+UseG1GC -XX:+ParallelRefProcEnabled -XX:MaxGCPauseMillis=200 -XX:+UnlockExperimentalVMOptions -XX:+AlwaysPreTouch -XX:G1NewSizePercent=30 -XX:G1MaxNewSizePercent=45 -XX:G1HeapRegionSize=8M -XX:G1ReservePercent=20 -XX:G1HeapWastePercent=5 -XX:G1MixedGCCountTarget=4 -XX:InitiatingHeapOccupancyPercent=15 -XX:G1MixedGCLiveThresholdPercent=90 -XX:G1RSetUpdatingPauseTimePercent=5 -XX:SurvivorRatio=32 -XX:+PerfDisableSharedMem -XX:MaxTenuringThreshold=1 -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xms256m -Xmx1400m
	Fabric Mods: 
		advancementinfo: AdvancementInfo 1.19.4-fabric0.76.0-1.3.1
			cloth-config: Cloth Config v10 10.0.96
				cloth-basic-math: cloth-basic-math 0.6.1
		c2me: Concurrent Chunk Management Engine 0.2.0+alpha.10.60
			c2me-base: Concurrent Chunk Management Engine (Base) 0.2.0+alpha.10.60
			c2me-client-uncapvd: Concurrent Chunk Management Engine (Client/Uncap View Distance) 0.2.0+alpha.10.60
			c2me-fixes-chunkio-threading-issues: Concurrent Chunk Management Engine (Fixes/Chunk IO/Threading Issues) 0.2.0+alpha.10.60
			c2me-fixes-general-threading-issues: Concurrent Chunk Management Engine (Fixes/General/Threading Issues) 0.2.0+alpha.10.60
			c2me-fixes-worldgen-threading-issues: Concurrent Chunk Management Engine (Fixes/WorldGen/Threading Issues) 0.2.0+alpha.10.60
			c2me-fixes-worldgen-vanilla-bugs: Concurrent Chunk Management Engine (Fixes/WorldGen/Vanilla Bugs) 0.2.0+alpha.10.60
			c2me-notickvd: Concurrent Chunk Management Engine (No Tick View Distance) 0.2.0+alpha.10.60
			c2me-opts-allocs: Concurrent Chunk Management Engine (Optimizations/Memory Allocations) 0.2.0+alpha.10.60
			c2me-opts-chunk-access: Concurrent Chunk Management Engine (Optimizations/Chunk Access) 0.2.0+alpha.10.60
			c2me-opts-chunkio: Concurrent Chunk Management Engine (Optimizations/Chunk IO) 0.2.0+alpha.10.60
			c2me-opts-math: Concurrent Chunk Management Engine (Optimizations/Math) 0.2.0+alpha.10.60
			c2me-opts-scheduling: Concurrent Chunk Management Engine (Optimizations/Scheduling) 0.2.0+alpha.10.60
			c2me-opts-worldgen-general: Concurrent Chunk Management Engine (Optimizations/General WorldGen) 0.2.0+alpha.10.60
			c2me-opts-worldgen-vanilla: Concurrent Chunk Management Engine (Optimizations/Vanilla WorldGen) 0.2.0+alpha.10.60
			c2me-rewrites-chunk-serializer: Concurrent Chunk Management Engine (Rewrites/Chunk Serializer) 0.2.0+alpha.10.60
			c2me-rewrites-chunkio: Concurrent Chunk Management Engine (Rewrites/Chunk IO) 0.2.0+alpha.10.60
			c2me-server-utils: Concurrent Chunk Management Engine (Server Utils) 0.2.0+alpha.10.60
			c2me-threading-chunkio: Concurrent Chunk Management Engine (Threading/WorldGen) 0.2.0+alpha.10.60
			c2me-threading-lighting: Concurrent Chunk Management Engine (Threading/Lighting) 0.2.0+alpha.10.60
			c2me-threading-worldgen: Concurrent Chunk Management Engine (Threading/WorldGen) 0.2.0+alpha.10.60
			com_electronwill_night-config_core: core 3.6.5
			com_electronwill_night-config_toml: toml 3.6.5
			com_ibm_async_asyncutil: asyncutil 0.1.0
			net_objecthunter_exp4j: exp4j 0.4.8
			org_threadly_threadly: threadly 7.0
		dynamicfps: Dynamic FPS 2.2.0
			com_moandjiezana_toml_toml4j: toml4j 0.7.2
		entityculling: EntityCulling-Fabric 1.6.2-mc1.19.4
		exordium: Exordium 1.1.1-mc1.19.4
		fabric-api: Fabric API 0.83.0+1.19.4
			fabric-api-base: Fabric API Base 0.4.27+1e9487d2f4
			fabric-api-lookup-api-v1: Fabric API Lookup API (v1) 1.6.28+504944c8f4
			fabric-biome-api-v1: Fabric Biome API (v1) 13.0.9+10ce000ff4
			fabric-block-api-v1: Fabric Block API (v1) 1.0.8+e022e5d1f4
			fabric-blockrenderlayer-v1: Fabric BlockRenderLayer Registration (v1) 1.1.37+10ce000ff4
			fabric-client-tags-api-v1: Fabric Client Tags 1.0.18+10ce000ff4
			fabric-command-api-v1: Fabric Command API (v1) 1.2.30+f71b366ff4
			fabric-command-api-v2: Fabric Command API (v2) 2.2.9+10ce000ff4
			fabric-commands-v0: Fabric Commands (v0) 0.2.47+df3654b3f4
			fabric-containers-v0: Fabric Containers (v0) 0.1.57+df3654b3f4
			fabric-content-registries-v0: Fabric Content Registries (v0) 3.5.12+10ce000ff4
			fabric-convention-tags-v1: Fabric Convention Tags 1.5.2+10ce000ff4
			fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.2.17+aeb40ebef4
			fabric-data-generation-api-v1: Fabric Data Generation API (v1) 11.4.3+10ce000ff4
			fabric-dimensions-v1: Fabric Dimensions API (v1) 2.1.48+10ce000ff4
			fabric-entity-events-v1: Fabric Entity Events (v1) 1.5.16+10ce000ff4
			fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.6.0+10ce000ff4
			fabric-events-lifecycle-v0: Fabric Events Lifecycle (v0) 0.2.55+df3654b3f4
			fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.35+a1ccd7bff4
			fabric-item-api-v1: Fabric Item API (v1) 2.1.20+10ce000ff4
			fabric-item-group-api-v1: Fabric Item Group API (v1) 3.0.8+10ce000ff4
			fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.35+504944c8f4
			fabric-keybindings-v0: Fabric Key Bindings (v0) 0.2.33+df3654b3f4
			fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 2.2.18+10ce000ff4
			fabric-loot-api-v2: Fabric Loot API (v2) 1.1.30+10ce000ff4
			fabric-loot-tables-v1: Fabric Loot Tables (v1) 1.1.34+9e7660c6f4
			fabric-message-api-v1: Fabric Message API (v1) 5.1.4+10ce000ff4
			fabric-mining-level-api-v1: Fabric Mining Level API (v1) 2.1.42+10ce000ff4
			fabric-models-v0: Fabric Models (v0) 0.3.33+10ce000ff4
			fabric-networking-api-v1: Fabric Networking API (v1) 1.3.4+10ce000ff4
			fabric-networking-v0: Fabric Networking (v0) 0.3.44+df3654b3f4
			fabric-object-builder-api-v1: Fabric Object Builder API (v1) 7.0.6+10ce000ff4
			fabric-particles-v1: Fabric Particles (v1) 1.0.26+10ce000ff4
			fabric-recipe-api-v1: Fabric Recipe API (v1) 1.0.11+10ce000ff4
			fabric-registry-sync-v0: Fabric Registry Sync (v0) 2.2.3+10ce000ff4
			fabric-renderer-api-v1: Fabric Renderer API (v1) 2.3.1+10ce000ff4
			fabric-renderer-indigo: Fabric Renderer - Indigo 1.3.1+10ce000ff4
			fabric-renderer-registries-v1: Fabric Renderer Registries (v1) 3.2.41+df3654b3f4
			fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.3.31+10ce000ff4
			fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 3.0.24+10ce000ff4
			fabric-rendering-v0: Fabric Rendering (v0) 1.1.44+df3654b3f4
			fabric-rendering-v1: Fabric Rendering (v1) 2.1.4+10ce000ff4
			fabric-resource-conditions-api-v1: Fabric Resource Conditions API (v1) 2.3.3+10ce000ff4
			fabric-resource-loader-v0: Fabric Resource Loader (v0) 0.11.5+5ade3c38f4
			fabric-screen-api-v1: Fabric Screen API (v1) 1.0.48+10ce000ff4
			fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.3.23+10ce000ff4
			fabric-sound-api-v1: Fabric Sound API (v1) 1.0.11+10ce000ff4
			fabric-transfer-api-v1: Fabric Transfer API (v1) 3.2.1+10ce000ff4
			fabric-transitive-access-wideners-v1: Fabric Transitive Access Wideners (v1) 3.2.0+10ce000ff4
		fabric-language-kotlin: Fabric Language Kotlin 1.9.4+kotlin.1.8.21
			org_jetbrains_kotlin_kotlin-reflect: kotlin-reflect 1.8.21
			org_jetbrains_kotlin_kotlin-stdlib: kotlin-stdlib 1.8.21
			org_jetbrains_kotlin_kotlin-stdlib-jdk7: kotlin-stdlib-jdk7 1.8.21
			org_jetbrains_kotlin_kotlin-stdlib-jdk8: kotlin-stdlib-jdk8 1.8.21
			org_jetbrains_kotlinx_atomicfu-jvm: atomicfu-jvm 0.20.2
			org_jetbrains_kotlinx_kotlinx-coroutines-core-jvm: kotlinx-coroutines-core-jvm 1.6.4
			org_jetbrains_kotlinx_kotlinx-coroutines-jdk8: kotlinx-coroutines-jdk8 1.6.4
			org_jetbrains_kotlinx_kotlinx-datetime-jvm: kotlinx-datetime-jvm 0.4.0
			org_jetbrains_kotlinx_kotlinx-serialization-cbor-jvm: kotlinx-serialization-cbor-jvm 1.5.0
			org_jetbrains_kotlinx_kotlinx-serialization-core-jvm: kotlinx-serialization-core-jvm 1.5.0
			org_jetbrains_kotlinx_kotlinx-serialization-json-jvm: kotlinx-serialization-json-jvm 1.5.0
		fabricloader: Fabric Loader 0.14.21
		fastload: Fast Load 3.4.0
			conditional-mixin: conditional mixin 0.3.2
			fastload-119-0-1-2-compat: Fastload 1.19-0-1-2 Compatibility 3.4.0
			fastload-1193-compat: Fastload 1.19.3 Compatibility 3.4.0
			fastload-1194-compat: Fastload 1.19.4 Compat 3.4.0
			fastload_fapi_forward: Fastload Fabric-api Forwarding 1.0.2
		ferritecore: FerriteCore 5.2.0
		ias: In-Game Account Switcher 8.0.1.2
		immediatelyfast: ImmediatelyFast 1.1.13
			net_lenni0451_reflect: Reflect 1.1.0
		iris: Iris 1.6.4
			io_github_douira_glsl-transformer: glsl-transformer 2.0.0-pre13
			org_anarres_jcpp: jcpp 1.4.14
			org_antlr_antlr4-runtime: antlr4-runtime 4.11.1
		java: OpenJDK 64-Bit Server VM 19
		krypton: Krypton 0.2.2
			com_velocitypowered_velocity-native: velocity-native 3.2.0-SNAPSHOT
		lithium: Lithium 0.11.1
		mambience: MAmbience 5.3.0+1.19.4
		minecraft: Minecraft 1.19.4
		modmenu: Mod Menu 6.2.2
		moreculling: More Culling 1.19.4-0.17.0
		nbteditor: NBT Editor 1.11.0
		nochatreports: No Chat Reports 1.19.4-v2.1.6
		optigui: OptiGUI 2.1.0-beta.3
			optiglue: OptiGlue 2.1.0-beta.3-mc.1.19.4
			org_apache_commons_commons-text: commons-text 1.10.0
			org_ini4j_ini4j: ini4j 0.5.4
		reeses-sodium-options: Reese's Sodium Options 1.5.0+mc1.19.4-build.72
		skinlayers: 3d Skin Layers 1.5.3-mc1.19.4
		sodium: Sodium 0.4.10+build.24
		sodium-extra: Sodium Extra 0.4.18+mc1.19.4-build.100
			caffeineconfig: CaffeineConfig 1.1.0+1.17
			crowdin-translate: CrowdinTranslate 1.4+1.19.3
		sound_physics_remastered: Sound Physics Remastered 1.19.4-1.1.1
		starlight: Starlight 1.1.1+fabric.ae22326
		visiblebarriers: Visible Barriers 2.0.1
			com_github_llamalad7_mixinextras: MixinExtras 0.2.0-beta.3
		vmp: Very Many Players 0.2.0+beta.7.73
		wurst: Wurst Client v7.35.1-MC1.19.4
	Loaded Shaderpack: (off)
	NEC status: No NEC detected
	Launched Version: 1.19.4
	Backend library: LWJGL version 3.3.1 SNAPSHOT
	Backend API: NVIDIA GeForce GTX 1070/PCIe/SSE2 GL version 3.2.0 NVIDIA 531.41, NVIDIA Corporation
	Window size: 1680x997
	GL Caps: Using framebuffer using OpenGL 3.2
	GL debug messages: 
	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'fabric'
	Type: Client (map_client.txt)
	Graphics mode: fast
	Resource Packs: fabric
	Current Language: en_us
	CPU: 12x AMD Ryzen 5 5600 6-Core Processor 
[16:26:15] [Render thread/ERROR]: Reported exception thrown!
net.minecraft.class_148: Manually triggered debug crash
	at net.minecraft.class_309.method_1474(class_309.java:509) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1574(class_310.java:1955) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1523(class_310.java:1180) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1514(class_310.java:801) ~[client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:237) ~[minecraft-1.19.4-client.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:468) ~[fabric-loader-0.14.21.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) ~[fabric-loader-0.14.21.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) ~[fabric-loader-0.14.21.jar:?]
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:88) ~[NewLaunch.jar:?]
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:126) ~[NewLaunch.jar:?]
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:71) ~[NewLaunch.jar:?]
Caused by: java.lang.Throwable: Manually triggered debug crash
	at net.minecraft.class_309.method_1474(class_309.java:506) ~[client-intermediary.jar:?]
	... 10 more
```

**Comment by itsTyrion** [comment]
E

**Comment by Alexander01998** [comment]
This bug has been fixed in the [Wurst 7.40](https://www.wurstclient.net/updates/wurst-7-40/) update.
